src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -54,25 +69,25 @@ def counter_names(self) -> list[str]:
         pass
 
     @abc.abstractmethod
-    def get_dedupe_value(self, data_packet: DataPacket[PacketT]) -> int:
+    def extract_dedupe_value(self, data_packet: DataPacket[DataPacketType]) -> int:
         """
         Extracts the deduplication value from a passed data packet.
-        TODO: This might belong on the `DataPacket` instead.
+        This is normally a timestamp.
```

Comment:
This raises more questions for me - when isn't it a timestamp? What else could it be? Does it matter so much what it is?

Reply:
I think I'd either remove that sentence entirely since the function is typed or expand upon it and list some things it could be

Reply:
what would be less confusing for you? do you have an example of what you'd be hoping to see?

right now, deduplication value doesn't feel like it's really clear what it should be used for, i'm hoping to add some clarity to that.

Reply:
I like the updated comment, thanks 👍